chirp/drivers/uv6r.py

```diff
@@ -853,7 +853,7 @@ def match_model(cls, filedata, filename):
         match_model = False
 
         # testing the file data size
-        if len(filedata) == 0x2008 or 0x2010:
+        if len(filedata) in (0x2008, 0x2010):
```

Comment:
I obviously prefer the latter, but these sorts of machine-generated changes just create noise in the git history for very little benefit, IMHO.

Reply:
The tool doesn't edit the code, I did it by hand.

If the purpose of that line was to accept as valid the length of 0x2008 and the length of 0x2010, then that code was broken because it accepted as valid also a length of zero:
```
>>> 0x2008 == 0x2008 or 0x2010
True
>>> 0x2010 == 0x2008 or 0x2010
8208
>>> 0x0000 == 0x2008 or 0x2010
8208
```

Reply:
> The tool doesn't edit the code, I did it by hand.

I know, I meant machine-generated complaints/suggestions.

> If the purpose of that line was to accept as valid the length of 0x2008 and the length of 0x2010, then that code was broken because it accepted as valid also a length of zero:

Ah, you're right. This looks very similar to some changes that "code hygiene" tools suggest that try to clean up "condition or condition or condition" to "value in [valid1, valid2, ...]". Apologies for jumping to that conclusion :)

--Dan